packages/react-components/src/components/table-output-component.tsx

```diff
@@ -653,4 +655,143 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
             });
         }
     }
+
+    async exportOutput(option: string): Promise<void> {
+        if (!this.gridApi || !this.columnApi) {
+            return;
+        }
+
+        this.closeExportMenu();
+
+        const traceUUID = this.props.traceId;
+        const tspClient = this.props.tspClient;
+        const outputId = this.props.outputDescriptor.id;
+        let fetchIndex = 0;
+        let totalLinesToFetch = this.gridApi.getDisplayedRowCount();
+        const linesArray = new Array<Line>();
+        const bufferSize = 100000;
```

Comment:
I suggest 50,000 so that it's closer to 5 seconds per iteration, but of course that depends on many factors, e.g. parsing speed in trace-server, network bandwidth, number of columns and length of the cell data...